index.html

```diff
@@ -2247,6 +2091,175 @@
   monitoring service integrity, load balancing, capacity planning, or billing); and,
 * has a contract in place with the [=actor=] which is consistent with the above limitations.
 
+## Recognition {#recognition}
+
+<dfn data-lt="recognize|recognized">Recognition</dfn> is the act of realising that a given [=identity=]
+corresponds to the same [=person=] as another [=identity=] which may have been
+observed either in another [=context=], or in the same [=context=] but at a
+different time.
```

Comment:
Down by "Cross-site recognition", we have

> when a [=site=] determines with high probability that a visit to the site comes from the same person as another visit to a *different* site.

The high probability bit probably belongs with recognition in general, rather than just with cross-site recognition... Maybe:

```suggestion
<dfn data-lt="recognize|recognized">Recognition</dfn> is the act of realising that a given [=identity=]
corresponds to the same [=person=] as another [=identity=] which may have been
observed either in another [=context=], or in the same [=context=] but at a
different time. [=Recognition=] can be probabilistic, if someone realises there's
a high probability that two [=identities=] correspond to the same [=person=],
even if they aren't certain.
```